### PR TITLE
Be able to do remote debugging in the project

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/debugging.rst [hunk 1]
```diff
@@ -141,6 +141,35 @@ If it is not already done, copy the file ``docker-compose.override.sample.yaml``
 ``docker-compose.override.yaml``.
 Be sure that the volumes for c2cgeoportal are uncommented.
 
+Remote dubugging
+................
+
+* In ``geoportal/requirements.txt`` uncomment.
```
Comment (arnaud-morvan) [inline doc/developer/debugging.rst]: uncomment ? seems something is missing here

File: doc/developer/debugging.rst [hunk 2]
```diff
@@ -141,6 +141,35 @@ If it is not already done, copy the file ``docker-compose.override.sample.yaml``
 ``docker-compose.override.yaml``.
 Be sure that the volumes for c2cgeoportal are uncommented.
 
+Remote dubugging
+................
+
```
Comment (arnaud-morvan) [inline doc/developer/debugging.rst]: Remote **debugging** using Visual Studio Code
.........................................................................

Or 

Remote **debugging**
.................................

Using Visual Studio Code

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl [hunk 3]
```diff
@@ -13,11 +13,15 @@ services:
...
-      # - '${PWD}/../c2cgeoportal:/opt/c2cgeoportal'
+      # - '${PWD}/../c2cgeoportal/commons/c2cgeoportal_gcommons:/opt/c2cgeoportal/commons/c2cgeoportal_commons'
+      # - '${PWD}/../c2cgeoportal/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal/geoportal/c2cgeoportal_geoportal'
+      # - '${PWD}/../c2cgeoportal/admin/c2cgeoportal_admin:/opt/c2cgeoportal/admin/c2cgeoportal_admin'
     environment:
       - GUNICORN_CMD_ARGS=--reload
       - C2CWSGIUTILS_CONFIG=/app/development.ini
       # - PRINT_URL=http://print:8080/print/
+    ports:
+      - 5678:5678
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl]: - 5678:5678  # If you are using Visual Studio Code
In my case Eclipse is listen to port 5678, and the python process connect to 172.17.0.1:5678

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl]: I don't understand your comment, finally it's the same, not?

Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl]: It is the inverse:
With VS Code: Python process listen port 5678, and VS Code is the client.
But Eclipse (or IntelliJ, PyCharm) listen port 5678 and python process is the client.

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.override.sample.yaml_tmpl]: OK, thanks :-)

**Pull request merged**